Create landing page for training materials (#157)
epiverse-trace/epiverse-trace.github.io: learn.qmd

---
format:
  html:
    toc: true
---

# Learning materials

> We organise our training materials in two curricula. The **Outbreak Analytics curriculum** aims to facilitate finding, choosing, and using R packages for outbreak analytics tasks to inform decision-making. The Research **Sofware curriculum** aims to extend our R toolset to produce open, reproducible and sustainable data analysis projects following good Research Sofware practices.

Author: ```suggestion
We organise our training materials in two curricula. The **Outbreak Analytics curriculum** aims to facilitate finding, choosing, and using R packages for outbreak analytics tasks to inform decision-making. The **Research Sofware curriculum** aims to extend our R toolset to produce open, reproducible and sustainable data analysis projects following good Research Sofware practices.
```

> We organise our training materials in two curricula. The **Outbreak Analytics curriculum** aims to facilitate finding, choosing, and using R packages for outbreak analytics tasks to inform decision-making. The Research **Sofware curriculum** aims to extend our R toolset to produce open, reproducible and sustainable data analysis projects following good Research Sofware practices.

Reviewer: May I assume that naming this and adding emphasis to the whole word `**Research Software Curriculum**` is too directed towards Research Software Engineering (RSE)? 

This aims to build on entry-level good scientific computing practices for researchers and outbreak data analysts. So it is fair to make a clear difference about this if needed. Happy to collect your name suggestions for this.

(a rename here will also imply an edit in the curricula subtitle lines below)

```suggestion
We organise our training materials in two curricula. The **Outbreak Analytics curriculum** aims to facilitate finding, choosing, and using R packages for outbreak analytics tasks to inform decision-making. The **Research Sofware curriculum** aims to extend our R toolset to produce open, reproducible and sustainable data analysis projects following good Research Sofware practices.
```

Reviewer 2: Styling already changed by @Bisaloo's suggestion.

For what it's worth, I did not style anything beyond the materials I was supplied with, to stay true to the provided copy. I only ported the Google Doc over as is.

> We address these goals with an e-Epi training kit, Tutorials,How-to guides, and Case studies:

Reviewer: For consistency with the order of listed elements:

```suggestion
We address these goals with an e-Epi training kit, Tutorials, Case studies, and How-to guides:
```

Reviewer 2: Included 👍

> We address these goals with an e-Epi training kit, Tutorials,How-to guides, and Case studies:

Author: ```suggestion
We address these goals with an e-Epi training kit, Tutorials, How-to guides, and Case studies:
```

| -------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| Análisis de Brotes y Modelamiento en Salud Pública | [{{< fa regular window-maximize title="Website">}}](https://epiverse-trace.github.io/epimodelac/) | [{{< fa brands github title="GitHub Repository" >}}](https://github.com/epiverse-trace/epimodelac/) | [{{< fa solid person-chalkboard title="Instructor view">}}](https://epiverse-trace.github.io/epimodelac/instructor/index.html) | [![Static Badge](https://img.shields.io/badge/pre-alpha-991b1b)](https://carpentries.github.io/lesson-development-training/lesson-design.html#iterative-development) |

> 

Reviewer: It would be cool to use [these features for tables](https://quarto.org/docs/authoring/tables.html#using-bootstrap-classes) that quarto provides.

```suggestion
: {.striped .hover}

```

### Tutorials in English

> | Tutorial                                                         | Site                                                                                                    | Repository                                                                                              | Instructor Schedule                                                                                                                 | Development stage                                                                                                                                                    |

Reviewer: Looking at how this looks on the cellphone (Please, see attached screenshot), I'm interested in finding ways to increase the width of the name column. 

Screenshot:
![image](https://github.com/epiverse-trace/epiverse-trace.github.io/assets/21230390/bed09dde-1f12-4759-ad4f-3686bcb1beb4)

I think:
- one could be to change the column name of the last column from `Development stage` to `Stage`. Any strong opinion on this?
- other, to combine the name and site columns, linking them directly in the name, as previously tested by @chartgerink in this PR.

Also, if we mix both alternatives we can 

```suggestion
| Tutorial                                                         | Site                                                                                                    | Repository                                                                                              | Instructor Schedule                                                                                                                 | Stage                                                                                                                                                    |
```

Reviewer: for the record, this is the outlook of the name and site columns mixed from a cellphone

![WhatsApp Image 2024-02-29 at 13 03 39_740f334f](https://github.com/epiverse-trace/epiverse-trace.github.io/assets/21230390/612d5a93-ac0f-4cf1-8d23-795d0365a5cb)

Author: https://quarto.org/docs/authoring/tables.html#column-widths

Reviewer 2: @avallecam please provide final input for the tables. I have changed them back and forth several times already, so in order to prevent any further issues, I will await your specific instructions for the final revision as part of this PR.

Reviewer: I agree with the final table outlook. I moved extra suggestions to separate issues to solve in other PRs #207 and #208

Para **Tutoriales en Español** recomendamos la [curricula del Software Carpentry en Español](https://software-carpentry.org/lessons/) con tutoriales sobre La terminal de Unix, Control de versiones con R, y R para análisis cientificos reproducible

## Your questions

Are you interested in participating in any of our workshops? Sign up for [our mailing list](https://data.org/initiatives/epiverse/our-community/), or follow us on Twitter at [Epiverse_TRACE](https://twitter.com/Epiverse_TRACE). You can also subscribe to [our blog feed](https://epiverse-trace.github.io/blog.html) and [take a look at our events calendar](https://view.monday.com/1267767805-fb40bb51c837164bf95532907829c3f8?r=euc1).

Are you interested in teaching these materials? For this and more questions about broader training topics, [start a discussion](https://github.com/orgs/epiverse-trace/discussions/new/choose).

> For questions about a specific learning material, go to its repository to [create an issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue).

Reviewer: From a chat with @lgbermeo, we found it valuable to add the list of workshops to this page. This will provide an idea to the readers of the amplitude and territoriality of the Epiverse-TRACE initiative and the possibilities of organising one of these locally. We expect to keep adding elements to this list after each training event. 

Also, as discussed in #193, we can optionally add links here to specific blog posts (like linking to these entries in [English](https://data.org/news/one-year-on-of-epiverse-trace-lac-exploring-the-data-science-and-health-ecosystem-in-latin-america-and-the-caribbean/) and [Spanish](https://data.org/news/un-ano-de-epiverse-trace-lac-explorando-el-ecosistema-de-ciencia-de-datos-y-salud-en-america-latina-y-el-caribe/))

```suggestion
For questions about specific learning material, go to its repository to [create an issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue).

# Our workshops

We used some of these materials in the following workshops:

- Curso Internacional: Análisis de Brotes y Modelamiento en Salud Pública https://epimodelac.com/, Bogotá (Colombia), December 4-8, 2023.
- Improving the reliability, usability and sustainability of code for epidemic analysis with R packages. IDDconf 2023, Ambleside, Cumbria, (UK), September 5-7, 2023.
- Curso-Taller: Herramientas de análisis y visualización de datos epidemiológicos para el reporte y la toma de decisiones en salud pública, Quibdó (Colombia), July 26-28, 2023.
- Curso-Taller: Teoría y práctica de respuesta a brotes y modelamiento matemático de epidemias, Bucaramanga (Colombia), April 27-29, 2023.
- Introductory training on Git and GitHub basics using Rstudio. London (UK), December 5, 2022.
- Taller de estudio de un brote y aplicación de la guía de estudio de brotes, Cali (Colombia), November 11-12, 2022.  
- Taller de Introducción a análisis de datos epidemiológicos en R y RStudio, Bogotá (Colombia),  October 24-25, 2023.
- Taller de Introducción al modelamiento y analítica de brotes y epidemias para informar la respuesta en salud pública, Bogotá (Colombia), October 24-25, 2023.
- Analizando un modelo básico de Zika, Cali (Colombia), October 15-16, 2022.
- Curso práctico en parámetros y modelos epidemiológicos en la respuesta a brotes y epidemias, Manizales (Colombia), September 1-2, 2022.
```

Reviewer 2: Could you please track this in a separate issue? I implemented the copy as suppliedl; this is new copy, which will need to be reviewed separately. Adding that review into this PR extends the time needed to merge this PR (and the mental space needed to keep track of all the discussions).

### How-to guides in English

| Material                                                                                                                  | Repository                                                               | Instructor schedule | Development stage                                                                                                                                      |
| ------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------ | ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------ |

> | [{{< fa regular window-maximize >}} How-to guides for Outbreak Analytics with R](https://epiverse-trace.github.io/howto/) | [{{< fa brands github >}} Repo](https://github.com/epiverse-trace/howto) | (empty)             | [![Static Badge](https://img.shields.io/badge/pre-alpha-991b1b)](https://carpentries.github.io/lesson-development-training/lesson-design.html#iterative-development) |

Reviewer: having an empty box could affect the final table outlook?

```suggestion
| [{{< fa regular window-maximize >}} How-to guides for Outbreak Analytics with R](https://epiverse-trace.github.io/howto/) | [{{< fa brands github >}} Repo](https://github.com/epiverse-trace/howto) |              | [![Static Badge](https://img.shields.io/badge/pre-alpha-991b1b)](https://carpentries.github.io/lesson-development-training/lesson-design.html#iterative-development) |
```

---
format:
  html:
    toc: true
---

# Learning materials

> Our trainings (1) facilitate how to find, choose, and use
> R packages for Outbreak Analytics tasks and (2) extending our R toolset
> to produce open, reproducible and sustainable data analysis projects
> following good Research Sofware practices. These two goals constitute
> the **Outbreak Analytics curriculum** and the **Research Sofware
> curriculum**.

Reviewer: Here I suggest a one-line edit to a [semantic line breaks](https://sembr.org/) text. Do you prefer to use this consistently across this document?

```suggestion
We organise our training materials in two curricula. The **Outbreak Analytics curriculum** aims to facilitate finding, choosing, and using R packages for outbreak analytics tasks to inform decision-making. The Research **Sofware curriculum** aims to extend our R toolset to produce open, reproducible and sustainable data analysis projects following good Research Sofware practices. 
```

  start your own journey.
- **Case studies** are real-life retrospective accounts of real projects
  delivered by users in the field. (coming soon)
- **How-to guides** are question-specific recipe-like scripts to show
  how multiple R packages can interoperate to get a solution.

Please note the development stage of the learning materials. A
"pre-alpha" tutorial is still being constructed. An
"alpha" tutorial is being taught by the original authors but
has not been fully tested.

## Outbreak Analytics Curriculum

These workshops focus on working with outbreak data using peer-reviewed interoperable open-source software like R packages to help the decision-making process in public health.

### e-Epi training kit en Español

El **e-Epi-training kit** es una estrategia para el aprendizaje en línea que permite un aprendizaje progresivo y flexible. Este kit incluye el curso virtual en ciencia de datos en salud pública y modelamiento de enfermedades infecciosas, el cual está dirigido a América Latina y el Caribe y cuenta con un enfoque de género. [Leer más detalles sobre la estrategia en línea](https://epiverse-trace.github.io/epi-training-kit/).

### Tutoriales en Español

> | Tutorial                                                                                                                              | Repositorio                                                                    | Agenda para instructoras/es                                                                                  | Fase de desarrollo                                                                                                                                                   |

Reviewer: @chartgerink could we keep isolated the name and site columns? replaced in https://github.com/epiverse-trace/epiverse-trace.github.io/pull/157/commits/dba3bd49d11faa0306719480c2c11439173eec57 but replacing the long links with the icon

the `name | site | repo | agenda | phase` outlook seems to keep all columns in all tables with similar widths.

### How-to guides in English

| Material                                                                                                                  | Repository                                                               | Instructor schedule | Development stage                                                                                                                                      |
| ------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------ | ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------ |

> | [{{< fa regular window-maximize >}} How-to guides for Outbreak Analytics with R](https://epiverse-trace.github.io/howto/) | [{{< fa brands github >}} Repo](https://github.com/epiverse-trace/howto) | (empty)             | experimental, WIP [https://github.com/epiverse-trace/howto?tab=readme-ov-file#howto](https://github.com/epiverse-trace/howto?tab=readme-ov-file#howto) |

Reviewer: could we replace this `(empty)` with empty content ` `? as in the [galaxy reference](https://training.galaxyproject.org/training-material/topics/covid19/)?

> | [{{< fa regular window-maximize >}} How-to guides for Outbreak Analytics with R](https://epiverse-trace.github.io/howto/) | [{{< fa brands github >}} Repo](https://github.com/epiverse-trace/howto) | (empty)             | experimental, WIP [https://github.com/epiverse-trace/howto?tab=readme-ov-file#howto](https://github.com/epiverse-trace/howto?tab=readme-ov-file#howto) |

Reviewer: I think we could reuse the carpentries lesson life cycle for homogeneity across materials displayed here.

```suggestion
| [{{< fa regular window-maximize >}} How-to guides for Outbreak Analytics with R](https://epiverse-trace.github.io/howto/) | [{{< fa brands github >}} Repo](https://github.com/epiverse-trace/howto) | (empty)             | [![Static Badge](https://img.shields.io/badge/pre-alpha-991b1b)](https://carpentries.github.io/lesson-development-training/lesson-design.html#iterative-development) |
```
